CONTRIBUTING.md

```diff
@@ -0,0 +1,45 @@
+# Contributing
+
+Thank you for your interest in contributing! We welcome all contributions no matter their size. Please read along to learn how to get started. More information can be found in each of the README.md as well as  https://docs.airswap.io/ about the individual repos.
+
+## Set up:
+First fork airswap-protocols repository and then clone the project. 
+
+Example:
+
+`git clone https://github.com/airswap/airswap-protocols`
+
+This project uses lerna to handle managing the mono-repo. Lerna is a tool for managing JavaScript projects with multiple packages.
```

Comment:
This project multiple many sub-repos found within `source` and `utils`. We use lerna to handle...

Reply:
This project multiple many sub-repos found within source and utils -> This project has multiple sub-repos found within `source` and `utils`.